ui/js/controllers/admin/exclusions_detail.js

```diff
@@ -0,0 +1,73 @@
+'use strict';
```

Comment:
I couldn't decide whether it was better to leave the ``exclusions_detail.js`` and ``exclusions_list.js`` controllers separate or merge them.  I'm really fine going either way, if you have a preference.  Same for the ``profiles_list.js`` and ``profiles_detail.js`` controllers.